app/utils/callback/lava.py

```diff
@@ -370,6 +370,11 @@ def add_boot(job_data, job_meta, lab_name, db_options,
     :type base_path: string
     :return ObjectId The boot document id.
     """
+    if job_data.get("status") not in (COMPLETE, INCOMPLETE):
```

Comment:
It would have been a bit better to add this test after the "Processing LAVA boot data..." message, to know which job is being skipped in the log.